content/200-orm/500-reference/325-prisma-config-reference.mdx

```diff
@@ -14,110 +14,193 @@ This feature is currently in [Early Access](/orm/more/releases#early-access) and
 
 :::
 
-The Prisma Config file is a TypeScript file that allows you to configure Prisma ORM. It accepts the following options:
+The `prisma.config.ts` file configures Prisma ORM using TypeScript. You can define your config in two ways:
```

Comment:
Let's phrase this differently: the config file configures the Prisma CLI - the commands you run to migrate, generate etc. 

It has no effect on Prisma when it runs in your app, ie the generated Prisma ORM client